install.sh

```diff
@@ -326,34 +332,34 @@ yes | crew install buildessential less most
 
 echo
 if [[ "${CREW_PREFIX}" != "/usr/local" ]]; then
-  echo "Since you have installed Chromebrew in a directory other than '/usr/local',"
-  echo "you need to run these commands to complete your installation:"
-  echo "echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc"
-  echo "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc"
-  echo "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc"
-  echo 'source ~/.bashrc'
+  echo -e "${YELLOW}Since you have installed Chromebrew in a directory other than '/usr/local',"
+  echo -e "${YELLOW}you need to run these commands to complete your installation:${RESET}"
+  echo -e "${BLUE}echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc"
+  echo -e "${BLUE}echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc"
+  echo -e "${BLUE}echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc"
+  echo -e "${BLUE}source ~/.bashrc${RESET}"
```

Comment:
Does this work?
```
  echo -e "${YELLOW}Since you have installed Chromebrew in a directory other than '/usr/local',"
  echo -e "you need to run these commands to complete your installation:${RESET}"
  echo -e "${BLUE}echo 'export CREW_PREFIX=${CREW_PREFIX}' >> ~/.bashrc"
  echo -e "echo 'export PATH=\"\${CREW_PREFIX}/bin:\${CREW_PREFIX}/sbin:\${PATH}\"' >> ~/.bashrc"
  echo -e "echo 'export LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}' >> ~/.bashrc"
  echo -e "source ~/.bashrc${RESET}"
```

Reply:
Just found a better solution
```bash
echo -e "${BLUE}To set the default PAGER environment variable to use less:
echo \"export PAGER='less'\" >> ~/.bashrc && . ~/.bashrc

Alternatively, you could use most.  Why settle for less, right?
echo \"export PAGER='most'\" >> ~/.bashrc && . ~/.bashrc

Below are some text editor suggestions.

To install 'nano', execute:
crew install nano

Or, to get an updated version of 'vim', execute:
crew install vim

You may wish to set the EDITOR environment variable for an editor default.

For example, to set 'nano' as the default editor, execute:
echo \"export EDITOR='nano'\" >> ~/.bashrc && . ~/.bashrc

To set 'vim' as the default editor, execute:
echo \"export EDITOR='vim'\" >> ~/.bashrc && . ~/.bashrc

${RESET}"
```